test/hotspot/jtreg/compiler/escapeAnalysis/TestScalarizeBailout.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2015, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
You should update the copyright year:
```suggestion
 * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```